rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/rules/CreateReportIndexRuleProvider.java

```diff
@@ -27,7 +27,7 @@
 @RuleMetadata(phase = ReportGenerationPhase.class)
 public class CreateReportIndexRuleProvider extends AbstractRuleProvider
 {
-    public static final String REPORT_INDEX = "Report Index";
+    public static final String REPORT_INDEX = "Dashboard";
```

Comment:
What is the Report list? Do you mean removal of Additional Reports from https://github.com/windup/windup/pull/1158 ?